### DISCUSSION TITLE
Authenticate user from another table in my DB.

### DISCUSSION THREAD

**Discussion by Jenish6108** [supabase/supabase#4819] [Questions]
Is there any other facility available by using that I can authenticate user from another table? I am having another table having custom authkey and user_id of user in it and I want to authenticate using that. How can I override the "auth.signIn()" function of supabase..??

**Comment by awalias** [comment]
hey @Jenish6108 thanks for the question, I'm having a little trouble understanding what you're trying to achieve. It is possible to write custom authorization logic in your [row level polices](https://supabase.com/docs/guides/auth/row-level-security), however the auth.signIn() method relies on users that exist in the auth.users table as per the [auth schema  ](https://github.com/supabase/supabase/blob/master/docker/volumes/db/init/01-auth-schema.sql)
